src/pages/trader-tools/common/_utility.js

```diff
@@ -0,0 +1,107 @@
+import validation from './_validation';
+
+const getMargin = (values) => {
```

Comment:
Some of these utility functions seem to be pip calculator specific and some seem to be shared with margin calculator. Could you help to keep specific ones in `_pip-calculator` and use the reusable ones for `_margin-calculator` as well? 🙏